recipes-devtools/pyelftools/python3-pyelftools_git.bb

```diff
@@ -1,6 +0,0 @@
-inherit setuptools3
-require python-pyelftools.inc
-
-RDEPENDS_${PN} = "python3 python3-contextlib python3-debugger python3-pprint"
-RDEPENDS_${PN}_class-native = "python3"
```

Comment:
Do you know why these dependencies are not part of the recipe in oe-core? I'm looking at this [file](https://git.openembedded.org/openembedded-core/tree/meta/recipes-devtools/python/python3-pyelftools_0.25.bb?h=dunfell&id=0a8cdaa90f4dd2d09b0b471dafd868a4dcad4ed3), is that right?

Reply:
Hmm... `python3-contextlib` is [a part](https://github.com/ni/openembedded-core/blob/nilrt/master/dunfell/meta/recipes-devtools/python/python3/python3-manifest.json#L230) of `python3-core` now, upon which pyelftools is automatically getting an `RDEPENDS`. So that is covered already.

I'm surprised to see that `python3-debugger` and `python3-pprint` are broken out (ibid.). And those seems to be real dependencies for pyelftools. So there might be a bug in the oe-core recipe. I think we're going to need someone to test pyelftools on a running system to be sure, though.

Reply:
Ok, how do you want to proceed with this commit? Merge and create a testing user story?

Reply:
I think so, yes. We need this commit (or a similar one) to get the images building enough to do that runtime testing. I'll hold off on trying to upstream this patchset to meta-measured until we have a confirmation that the oe-core recipe doesn't regress.